data/hai outfits.txt

```diff
@@ -114,7 +114,7 @@ outfit "Railgun"
 		"hit force" 44
 		"stream"
 		"submunition" "rslug" 1
-	description "The Hai created this weapon for their Flea drones, providing mobile long range support for their close quarters Pond Strider carrier. This particular incarnation of a rail weapon imparts a massive static electric charge to the projectile's flux capacitor, and motivates the slug with electric repulsion. Once in flight, the projectile's charge will bend trajectory very slightly, and allows it to damage the hull a little bit through shields."
+	description "The Hai created this miniaturized incarnation of the classic railgun for their Flea drones, providing mobile long range support for the close quarters Pond Strider carrier."
```

Comment:
the Particle Cannon is, by it's description, a de-facto rail gun.
the Proton Gun was an attempt to make a cheap Particle Cannon (so, also a Rail Gun).